notebooks/curation_api/python/src/dataset.py

```diff
@@ -57,51 +57,31 @@ def delete_dataset(collection_id: str, dataset_id: str):
         success(logger, success_message)
 
 
-def get_download_links_for_assets(collection_id: str, dataset_id: str):
+def download_assets(datasets: list, log_version_id: bool = False):
```

Comment:
I know this is pedantic but I think this is a strictly worse method name. The method doesn't download the assets. It retrieves urls for the assets.

Reply:
`get_asset_urls`?

Reply:
whoops